lib/Conch/Route.pm

```diff
@@ -37,34 +37,49 @@ Set up the full route structure
 =cut
 
 sub all_routes {
-	my $unsecured = shift;	# this is the base routing object
+	my $root = shift;	# this is the base routing object
 	my $features = shift || {};
 
+	# provides a route to chain to that first checks the user is a global admin.
+	$root->add_shortcut(require_global_admin => sub {
+		my ($r, $path) = @_;
+		$r->any(sub {
+			my $c = shift;
+			return $c->status( 403, { error => 'Login required' } )
```

Comment:
does conch-shell or conch-api rely on 403 vs 401? (this change is just a translation of what was already in Conch::Controller::User::revoke_user_tokens.)

Reply:
The shell has a helper to handle http responses: https://github.com/joyent/conch-shell/blob/master/pkg/conch/errors.go#L63

A change from a 403 to a 401 will change the response value to a different error but it shouldn't be an issue. The only substantive change will be to the text the user sees, since all the functions bail when any error occurs.

In this specific case, I think an "unauthorized" message will be more descriptive and clear than "forbidden".